server/src/main/java/com/google/musicanalysis/site/YoutubeGenres.java

```diff
@@ -0,0 +1,14 @@
+package com.google.musicanalysis.site;
```

Comment:
not sure if this is the correct area to put this java class
I added it here: `server/src/main/java/com/google/musicanalysis/site/YoutubeGenres.java`